### PR TITLE
Fix sticky toolbar for editor3

### PR DESCRIPTION
**Pull request by tomaskikutis**
SDESK-5765

### CODE REVIEW

**Comment by tomaskikutis** [comment]
Yep, forgot about it. Added it now.

**Review by petrjasek — APPROVED** [review]

**Comment by MarkLark86** [comment]
@petrjasek This commit https://github.com/superdesk/superdesk-client-core/commit/e98aa703da8ef026c4ec99b104faaea045c76ad2 will need to be cherry-picked as well into any version this is back-ported to (to remove the `superdesk-planning` submodule)

File: scripts/apps/authoring/views/article-edit.html [hunk 1]
```diff
@@ -641,7 +641,7 @@
         data-single-line="field.field_options.single"
         data-value="item.extra[field._id]"
         data-bind-to-value="compareView"
-        data-scroll-container=".page-content-container"
+        data-scroll-container=".ppage-content-container--scrollable"
```
Comment (petrjasek) [inline scripts/apps/authoring/views/article-edit.html:644]: seems like typo `.PPage`

Comment (tomaskikutis) [reply scripts/apps/authoring/views/article-edit.html:644]: here's the fix https://github.com/superdesk/superdesk-client-core/pull/3867

**Pull request merged**